cpp/lib/include/Zotero.h

```diff
@@ -301,8 +321,19 @@ class MarcFormatHandler final : public FormatHandler {
                              std::string * const publication_title, std::string * const abbreviated_publication_title,
                              std::string * const website_title);
 
-    void extractCustomNodeParameters(std::shared_ptr<const JSON::JSONNode> custom_node,
+    void ExtractCustomNodeParameters(std::shared_ptr<const JSON::JSONNode> custom_node,
                                      struct CustomNodeParameters * const custom_node_parameters);
+
+    void ExtractItemParameters(std::shared_ptr<const JSON::ObjectNode> custom_node,
```

Comment:
inconsistent naming: first parameter "custom_node" should be "object_node" like in Zotero.cc!